src/Entity/EdgeEntityTypeInterface.php

```diff
@@ -34,4 +34,14 @@ interface EdgeEntityTypeInterface extends EntityTypeInterface {
    */
   public function getQueryClass(): string;
 
+  /**
+   * The default theme used by the entity view builder to render this entity.
+   *
+   * @return string
+   *   Name of the theme.
+   *
+   * @see \Drupal\apigee_edge\Entity\EdgeEntityViewBuilder::getBuildDefaults()
+   */
+  public function getDefaultTheme(): string;
```

Comment:
This is a BC break because it introduces a new public method, but I believe it worth it. See in EdgeEntityViewBuilder why.